Source/WebCore/html/HTMLTrackElement.h

```diff
@@ -72,7 +72,7 @@ class HTMLTrackElement final : public HTMLElement, public ActiveDOMObject, publi
     RefPtr<HTMLMediaElement> mediaElement() const;
     const AtomString& mediaElementCrossOriginAttribute() const;
 
-    void scheduleTask(Function<void()>&&);
+    void scheduleTask(Function<void(HTMLTrackElement&)>&&);
```

Comment:
Consider making this a static method if you're now passing in the track.

Reply:
It cannot be made static, the implementation relies on `this`.